plugins/chat/assets/javascripts/discourse/components/chat-channel-message-emoji-picker.gjs

```diff
@@ -13,6 +13,10 @@ export default class ChatChannelMessageEmojiPicker extends Component {
   context = "chat-channel-message";
 
   listenToBodyScroll = modifier(() => {
+    if (!this.site.mobileView) {
```

Comment:
Why is this change needed? Is the `next()` change somehow triggering a `scroll` event? 🤔 

Also... closing the picker on scroll is a deliberate feature. Are we really ok to drop it?